supporting_components/graph.py

```diff
@@ -45,6 +45,8 @@ def __init__(self, graph: "Graph", label=None, graph_label=None):
         self.label = label
         self.graph_label = graph_label
         self._incidence = {}
+        self.twin_degree = None
```

Comment:
Moet deze property behouden blijven als je een graph copied ? (r. 434 supporting_components/graph.py)
Hij staat niet in het lijstje van Vertex properties als je een graph copied.

Reply:
Heb hem toegevoegd, dit kan natuurlijk voorkomen